recipes/shapelib/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class ShapelibConan(ConanFile):
+    name = "shapelib"
+    description = "C library for reading and writing ESRI Shapefiles"
+    license = "LGPL-2.0-or-later"
+    topics = ("conan", "shapelib", "osgeo", "shapefile", "esri", "geospatial")
+    homepage = "https://github.com/OSGeo/shapelib"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = 'CMakeLists.txt'
+    generators = 'cmake'
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "set(BUILD_TEST ON)",
+                              "option(BUILD_TEST \"\" ON)")
```

Comment:
```suggestion
                              "")
```

Why instead you just don't remove it?

Reply:
Mmhhhh, good point !

Therefore, I'll also remove `cmake.definitions["BUILD_TEST"] = False`